src/simtools/applications/simulate_prod.py

```diff
@@ -122,6 +122,17 @@ def _parse(description=None):
         required=False,
         default=False,
     )
+    config.parser.add_argument(
```

Comment:
Wondering if the default shouldn't be 'true'? We will run all major productions on the grid with the single core requirement.

Reply:
I don't know. On the grid we will always run with a configuration file which can always have this kind of setting explicit (which is a small bonus as well). However, when running locally or during testing, it's easier to keep the configuration short and use as many cores as possible to speed things up.

It's not a strong opinion, but I think it makes more sense this way.